codecov.yml

```diff
@@ -6,8 +6,14 @@ coverage:
       default: false
       python:
         paths:
-        - "src/sentry"
+        - "src/sentry/**/*.py"
+        ignore:
```

Comment:
I'm not sure if this is a valid option when nested like this, but it's also unneeded since `paths` includes only *.py files now